### PR TITLE
improve flask dev startup by using `flask run`

### PR DESCRIPTION
**Pull request by asottile-sentry**

### CODE REVIEW

File: flask/run.sh [hunk 1]
```diff
@@ -5,18 +5,15 @@ set -e
...
     python3 -m venv ./venv
 fi
 source venv/bin/activate
 pip3 install -r requirements.txt
 
 function cleanup {
-  stop.sh python3 $LOCAL_PORT 
-  if [[ "$VIRTUAL_ENV" != "" ]]; then
-    deactivate
-  fi
```
[the remark this answers is not in the document]
Comment (realkosty) [reply flask/run.sh:18]: oh right, I forget that all it does is set env variables
nice video!!

File: flask/run.sh [hunk 2]
```diff
@@ -5,18 +5,15 @@ set -e
...
+  stop.sh python3 $LOCAL_PORT
 }
 trap cleanup EXIT
 
-PORT=$LOCAL_PORT python3 run.py
+flask run --port $LOCAL_PORT
```
Comment (realkosty) [inline flask/run.sh:19]: should we then also get rid of `run.py` and [this section](https://github.com/sentry-demos/empower/blob/master/flask/src/main.py#L238C11-L244) now that they are no longer used?

Comment (asottile-sentry) [reply flask/run.sh:19]: ah I did delete run.py but I'll delete that too

**Review by realkosty — APPROVED** [review]
thank you!

**Pull request merged**